loop_enable: False
speed_control: False
initial_lookahead_distance : 0.4
initial_velocity : 0.4
controller_rate : 10
goal_threshold: 0.5
intermediate_goal_threshold: 0.15

emotional_state : "happy"

sleepy_vel_min: 0.1
sleepy_vel_max: 0.55
ramp_rate: 0.25
execution_delay_min: 3
execution_delay_max: 6

straight_lookahead_dist : 3.0
turning_lookahead_dist : 0.5
straight_vel : 2.0
turning_vel : 0.5
turn_window_min : 3.5
turn_window_max : 4.5
max_x_deviation : 2.5
